osquery/core/posix/process.cpp

```diff
@@ -177,13 +177,21 @@ std::shared_ptr<PlatformProcess> PlatformProcess::launchPythonScript(
     const std::string& args) {
   std::shared_ptr<PlatformProcess> process;
   std::string argv;
-
-  if (!isPlatform(PlatformType::TYPE_FREEBSD)) {
-    argv = "/usr/local/osquery/bin/python " + args;
+  std::string osquery_path;
+
+  char const* tmp = getenv("OSQUERY_DEPS");
```

Comment:
Replaced 9556a4d18f602c792a4207efcde5ae982e749dea with b301bbe057a0a82401846a171fc27414a8aab7d5 to use `getEnvVar()` instead of `getenv()`.